src/coreclr/jit/emitarm64.cpp

```diff
@@ -9831,16 +9847,59 @@ void emitter::emitIns_R_R_R_R(
             fmt = IF_NONE;
             break;
 
+        case INS_sve_cmpeq:
+        case INS_sve_cmpgt:
+        case INS_sve_cmpge:
+        case INS_sve_cmphi:
+        case INS_sve_cmphs:
+        case INS_sve_cmpne:
+        case INS_sve_cmple:
+        case INS_sve_cmplo:
+        case INS_sve_cmpls:
+        case INS_sve_cmplt:
+            assert(insOptsScalableSimple(opt));
+            assert(isPredicateRegister(reg1));    // DDDD
+            assert(isLowPredicateRegister(reg2)); // ggg
+            assert(isVectorRegister(reg3));       // mmmmm
+            assert(isVectorRegister(reg4));       // nnnnn
+            assert(isScalableVectorSize(attr));   // xx
+            fmt = IF_SVE_CX_4A;
+            break;
+
         default:
             unreached();
             break;
     }
     assert(fmt != IF_NONE);
 
+    // Use aliases.
+    switch (ins)
+    {
+        case INS_sve_cmple:
+            std::swap(reg3, reg4);
+            ins = INS_sve_cmpge;
```

Comment:
```suggestion
            ins = INS_sve_cmpgt;
```

Reply:
This confused me for a while. When inverting a comparison the equals state needs switching (`le` -> `gt`). But here we want the result to not change. `gt` would give the wrong result when the two inputs are the same. 

https://docsmirror.github.io/A64/2023-06/cmple_cmpeq_p_p_zz.html

`a LE b` -> `b GE a`.